nuget/runners/nunit.console-runner-with-extensions.nuspec

```diff
@@ -26,14 +26,14 @@
     <releaseNotes>This release uses the latest version of the TeamCityEventListener extension.</releaseNotes>
     <language>en-US</language>
     <tags>nunit test testing tdd runner</tags>
-    <copyright>Copyright (c) 2016 Charlie Poole</copyright>
+    <copyright>Copyright (c) 2017 Charlie Poole</copyright>
     <dependencies>
       <group>
-        <dependency id="NUnit.ConsoleRunner" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.VSProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2ResultWriter" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2Driver" version="$version$" />
+          <dependency id="NUnit.ConsoleRunner" version="3.5.0" />
```

Comment:
The script needs to be updated to usher in the new era of diverse versions in the packages. This was mentioned in some earlier issue, but I can't remember where. Essentially, we need a list of all the extensions to be included together with the version of each of them. If you really need to get the release out today, then I suggest hard-coding the versions and dealing with this problem later on. But do you?